src/shared/components/annotation/CivicCard.spec.tsx

```diff
@@ -35,7 +35,7 @@ describe('CivicCard', () => {
 
     it('should not have variant', () => {
         const wrapper = shallow(<CivicCard {...props}/>);
-        expect(wrapper.find('.civic-card-variant')).to.have.length(0);
+        expect(wrapper.find('.civic-card-variant')).to.have.length(1);
```

Comment:
So, the unit test definition claims that `it should not have variant`, but then we expect to have 1 `civic-card-variant`. Seems a bit contradicting to me.

Reply:
Now, with the new implementation, `civic-card-variant` is created when there are no variants, only with the `civic-card-variant-description summary` item. However, other items of this part like `civic-card-variant-header`, `civic-card-variant-name` and `civic-card-variant-entry-types` are not created, so I am going to add this three parts as the test to check if there is a variant or not, I think it would make more sense.